app/python/synchrona/ad-synchrona-14-leds.sh

```diff
@@ -0,0 +1,94 @@
+#!bin/bash
+
+HMC7044_STATUS_PATH="/sys/kernel/debug/iio/iio:device0/status"
+AD9545_STATUS_PATH="/sys/kernel/debug/clk/PLL1/PLL1"
+
+RED_LED1=12
+GREEN_LED1=16
+
+#export all led gpios and set as output
+echo $RED_LED1 > /sys/class/gpio/export
+echo "out" > "/sys/class/gpio/gpio$RED_LED1/direction"
```

Comment:
nit: this does not hurt but these pins should be already be set to out by the pinctrl... Note that this can actually be obfuscating a bug in the devicetree. That was the case for the vcxo selection